CefSharp.BrowserSubprocess.Core/JavascriptRootObjectWrapper.h

```diff
@@ -17,6 +17,8 @@ using namespace CefSharp::Internals::Async;
 
 namespace CefSharp
 {
+    ref class CefBrowserWrapper;
```

Comment:
I dislike using forward declarations. Hopefully with a little refactoring we can do away with this.